lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -118,6 +115,56 @@ public IObserver<T> GetRemoteObserver(IPEndPoint remoteEndpoint)
         }
 
         /// <summary>
+        /// Returns an IRemoteObserver used to send messages to the remote host at
+        /// the specified IPEndpoint.
+        /// </summary>
+        /// <param name="remoteEndpoint">The IPEndpoint of the remote host</param>
+        /// <returns>An IRemoteObserver used to send messages to the remote host</returns>
+        public IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> remoteEndpoint)
```

Comment:
It is not part of the REEF API. Hence, it *should* be safe to make `internal` or mark as `[Private]` and do as we see fit with it.